src/utils/getOffsetGMT.ts

```diff
@@ -0,0 +1,9 @@
+export default function getOffsetGMT() {
+  function isSingleDigit(n: number) {
+    return `${n < 10 ? '0' : ''}${n}`;
+  }
+  let offset = new Date().getTimezoneOffset();
```

Comment:
Can 7th line be moved here?  ( `Math.abs(new Date().getTimezoneOffset())` ) or does it change the outcome?